src/main/java/com/synopsys/integration/alert/web/actions/SystemActions.java

```diff
@@ -91,10 +91,10 @@ public boolean isSystemInitialized() {
 
     public FieldModel getCurrentSystemSetup() {
         final Map<String, FieldValueModel> valueMap = new HashMap<>();
-        FieldModel model = new FieldModel(settingsDescriptorKey.getUniversalKey(), ConfigContextEnum.GLOBAL.name(), valueMap);
+        FieldModel model = new FieldModel(settingsUtility.getSettingsName(), ConfigContextEnum.GLOBAL.name(), valueMap);
```

Comment:
If we're not gonna keep the descriptor key here, then why aren't we using the Utility to get the FieldModel?